PennyPanX/git-cooperation
# Desc: Determine whether the device is FSD according to unitmodel

Card Number: SS-10062
Desc: Determine whether the device is FSD according to unitmodel
Testing Procedures:

Login;
Add a "Status Display" instance;
Check layouts in main menu.
Link:https://emersonfoodretail.atlassian.net/browse/SS-10062
